test_scripts/Smoke/HeartBeat/ATF_HeartBeat_no_heartbeat_v2_protocol.lua

```diff
@@ -0,0 +1,99 @@
+--  Requirement summary:
+--  [Services]: SDL must support Heartbeat over protocol v3 or higher
+--
+--  Description:
+--  Check that no heartbeat occurs if App uses v2 protocol version.
+
+--  1. Used precondition
+--  SDL, HMI are running.
+--  Mobile device is connected.
+--  HeartBeatTimeout = 5000
+--
+--  2. Performed steps
+--  Start SPT, select transport, specify protocols = 2
+--  Wait 15 sec.
+--
+--  Expected behavior:
+--  1. App has successfully registered.
+--  2. App is still registered, no unexpected disconnect occurs.
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local constants = require('protocol_handler/ford_protocol_constants')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
```

Comment:
@LitvinenkoIra, will be good to updated INI file with HeartBeatTimeout = 5000. Please don't forget or backup file and restore it in the end of scripts.